artifactory_cleanup/artifactorycleanup.py

```diff
@@ -72,12 +77,14 @@ def _delete(artifact):
             try:
                 artifacts_size = sum([x["size"] for x in artifacts_to_remove])
                 print("Summary size: {}".format(artifacts_size))
-                yield CleanupSummary(
+                summary = CleanupSummary(
                     policy_name=policy.name,
                     artifacts_size=artifacts_size,
                     artifacts_removed=len(artifacts_to_remove),
                 )
-
+                if self.output_format == "json" and self.output_artifacts:
+                    summary.removed_artifacts_list = artifacts_to_remove
+                yield summary
```

Comment:
Let's always pass the list of removed artifact, the upper layard (output) will know when to save it or not.
```suggestion
                yield summary
```